### DISCUSSION TITLE
How to Stop Seeding while Downloading Via qBittorrent

### DISCUSSION THREAD

**Discussion by MotherFatherGoodManner** [anasty17/mirror-leech-telegram-bot#1047] [General]
Hi,

I was wondering is there any way to stop the seeding of files, at the time of downloading a torrent via qBittorrent.
I know about the default seeding option of the Repo, I turned that off,

but this seeding while downloading eating all credit of my gcp vps, (as gcp charging for sending bandwidth outside google)

**Comment by anasty17** [comment]
U mean upload not seed! U still downloading.
In qBittorrent we can't limit upload speed zero,  u can limit it from qBittorrent config to slow speed but this will result to slow download speed and if upload speed is zero then download speed zero.

Use aria2c!

**Comment by MotherFatherGoodManner** [comment]
Yes I'm talking about the Upload while downloading file.

Seems like I have to move to aria2c cause qbit is seeding while downloading and it consuming my gcp credit very fast.

**Reply by anasty17** [reply]
It's up to u
